packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/permissions/PermissionsModule.java

```diff
@@ -175,7 +175,7 @@ public boolean onRequestPermissionsResult(
       mCallbacks.get(requestCode).invoke(grantResults, getPermissionAwareActivity());
       mCallbacks.remove(requestCode);
```

Comment:
Instead add a null-check here.
```suggestion
      Callback callback = mCallbacks.get(requestCode);
      if (callback != null) {
         callback.invoke(grantResults, getPermissionAwareActivity());
         mCallbacks.remove(requestCode);
      } else {
         // Log warning. It's unexpected we're getting a response here we didn't ask for.
      }
```